TestCases/compliance-level-3/1121-feel-years-and-months-duration-function/1121-feel-years-and-months-duration-function.dmn

```diff
@@ -353,56 +351,38 @@
       <dmn:text>years and months duration([],[])</dmn:text>
     </dmn:literalExpression>
   </dmn:decision>
-  <dmn:decision id="_JUuHUclGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_ErrorCase_31">
-    <dmn:description>Tests FEEL expression: 'years and months duration(date(&quot;2013-08-24&quot;), date and time(&quot;2017-12-15T00:59:59&quot;))' and expects result: 'null (null)'</dmn:description>
-    <dmn:question>Result of FEEL expression 'years and months duration(date(&quot;2013-08-24&quot;), date and time(&quot;2017-12-15T00:59:59&quot;))'?</dmn:question>
-    <dmn:allowedAnswers>null (null)</dmn:allowedAnswers>
-    <dmn:variable id="_JUuHUslGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_ErrorCase_31" typeRef="ns1:tfeel-years-and-months-duration-function_31_Result"/>
-    <dmn:literalExpression id="_JUuHUMlGEeeAGbzZrHaKcQ">
-      <dmn:text>years and months duration(date(&quot;2013-08-24&quot;), date and time(&quot;2017-12-15T00:59:59&quot;))</dmn:text>
-    </dmn:literalExpression>
-  </dmn:decision>
-  <dmn:decision id="_JUvVcclGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_ErrorCase_32">
-    <dmn:description>Tests FEEL expression: 'years and months duration(date and time(&quot;2017-02-28T23:59:59&quot;), date(&quot;2019-07-23&quot;) )' and expects result: 'null (null)'</dmn:description>
-    <dmn:question>Result of FEEL expression 'years and months duration(date and time(&quot;2017-02-28T23:59:59&quot;), date(&quot;2019-07-23&quot;) )'?</dmn:question>
-    <dmn:allowedAnswers>null (null)</dmn:allowedAnswers>
-    <dmn:variable id="_JUv8gMlGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_ErrorCase_32" typeRef="ns1:tfeel-years-and-months-duration-function_32_Result"/>
-    <dmn:literalExpression id="_JUvVcMlGEeeAGbzZrHaKcQ">
-      <dmn:text>years and months duration(date and time(&quot;2017-02-28T23:59:59&quot;), date(&quot;2019-07-23&quot;) )</dmn:text>
-    </dmn:literalExpression>
-  </dmn:decision>
-  <dmn:decision id="_JUxKoclGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_33">
+  <dmn:decision id="_JUxKoclGEeeAGbzZrHaKcQ" name="feel-years-and-months-duration-function_31">
```

Comment:
@kurobako can you please double check this commit? the diff does not seem consistent...

Reply:
@etirelli diff might look weird, but I don't actually see what's wrong with my PR. I removed tests 31 and 32 and renamed following tests so that there would be no gaps in sequence: 33 -> 31, 34 -> 32, 35 -> 33, 36 -> 34

Reply:
@kurobako it is best to not rename tests because it affects the result files submitted by vendors. Having gaps in the numbers is not a problem.